Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -6930,6 +6928,89 @@ def initialize_instance(args: Dict[str, str], params: Dict[str, str]):
         XPATH_RULEBASE = f"/config/devices/entry[@name=\'localhost.localdomain\']/vsys/entry[@name=\'{VSYS}\']/"
 
 
+def panorama_upload_content_update_file_command(args: dict):
+    category = args.get('category')
+    entry_id = args.get('entryID')
+    file_path = demisto.getFilePath(entry_id)['path']
+    file_name = demisto.getFilePath(entry_id)['name']
+
+    try:
+        shutil.copy(file_path, file_name)
+    except Exception:
+        raise Exception('Failed to prepare file for upload.')
+
+    with open(file_name, 'rb') as file:
+        files = {'file': file}
```

Comment:
What is the purpose of this variable? unclear from the name IMO

Reply:
DONE